modules/serverpod_auth/serverpod_auth_server/lib/src/firebase/auth/auth_http_client.dart

```diff
@@ -0,0 +1,11 @@
+import 'package:http/http.dart';
+
+/// Base Auth Http Client
+class AuthBaseClient extends BaseClient {
```

Comment:
Question: The `AuthBaseClient` does not seem to add much value, could it be removed?